crm-data-components/src/app/extensions/package.json

```diff
@@ -14,8 +14,5 @@
   "dependencies": {
     "@hubspot/ui-extensions": "latest",
     "react": "^18.2.0"
-  },
-  "devDependencies": {
-    "@hubspot/ui-extensions-dev-server": "latest"
```

Comment:
Wow, this is a blast from the past

Reply:
@joe-yeager I found references to this in recently launched customer code! This is getting duplicated somehow, and I have no clue where it's coming from. So I wanted to kill these so they don't become a potential source of spread

Reply:
Oh yeah, absolutely!  Just made me nostalgic about the dev-servers humble beginnings